openandroidinstaller/assets/configs/a7y17lte.yaml

```diff
@@ -0,0 +1,28 @@
+metadata:
+  maintainer: Tobias Sterbak (tsterbak)
```

Comment:
Oh and you should also put your name / nickname here, as you are the maintainer and creator of the config :D